code/frontend/src/components/VideoDetails.js

```diff
@@ -1,8 +1,37 @@
-// Get the reward points and name elements
-const rewardPointsElement = document.querySelector('#reward-points');
-const nameElement = document.querySelector('#name');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '../styles/VideoDetails.css'
 
-// Update the reward points and name elements
-rewardPointsElement.textContent = 100;
-nameElement.textContent = 'Paridhi Talwar';
+const VideoDetails = () => {
+  return (
+    <div className="container">
+      <button type="button">
+        <img src="logo.png" alt="Logo" />
+      </button>
+      <div className="profile-card">
+        <img src="profile-pic.jpg" alt="Profile picture" />
+        <h4>Reward Points: <span id="reward-points">100</span></h4>
+      </div>
+      <div className="video-player">
+        <h1>Video Player</h1>
+        <iframe
+                width="420"
+                height="315"
+                src="https://www.youtube.com/embed/SqcY0GlETPk"
```

Comment:
could you change this so that the video url is not hardcoded but coming from a variable?
Like 
`const currentVideo = "https://www.youtube.com/....."`

and in your `<iframe src={currentVideo}>`